test/command/Script/executeFunctionWithPrint.js

```diff
@@ -0,0 +1,78 @@
+import ScriptCommand from "../../../renderEngine/Command/ScriptCommand.js";
+import ContextBase from "../../../renderEngine/Context/ContextBase.js";
+import CancellationToken from "../../../renderEngine/Cancellation/CancellationToken.js";
+import PrintCommand from "../../../renderEngine/Command/PrintCommand.js";
+
+import JsonSource from "../../../renderEngine/Source/JsonSource.js";
+const context = new ContextBase();
+context.cancellation = new CancellationToken();
+const scriptIl = {
+  $type: "script",
+  core: "script",
+  name: "script",
+  runType: "AtServer",
+  language: "javascript",
+  content: `  function add(a, b) {
+
+    return Number(a) + Number(b);
+  }
+  function subtract(a, b){
+    return a - b;
+  }
+ this is a test string
+  const divide = (e, f) => {
+    return e / f;
+  }`,
+};
+const command = new ScriptCommand(scriptIl);
+await command.executeAsync(context);
```

Comment:
please add execute all command in same time.  and use `Promise.all()` in wait for result